packages/py-api/py_api/models/hackathon_participants_models.py

```diff
@@ -0,0 +1,19 @@
+from typing import Literal, Optional
+
+from pydantic import BaseModel
+
+TSHIRT_SIZE = Literal["S", "M", "L"]
+
+
+class NewParticipant(BaseModel):
+    first_name: str
+    last_name: str
+    email: str
+    tshirt_size: TSHIRT_SIZE
```

Comment:
Add fields:
* verified: false. This field should not be required but with every post request, it should be added to the db.
* team_name: optional. If passed take the passed value, if not passed -> None or empty str.